forms-flow-idm/realm-exports/Client based auth.json

```diff
@@ -43,9 +43,8 @@
             "realm-management":[
                "view-users",
                "query-users",
-               "manage-clients",
                "query-clients",
-               "manage-users"
```

Comment:
any reason to remove `manage-users`?   there is a `locale` endpoint that needs manage-users as client roles in the service account to update the locale user attribute. Please confirm @sumesh-aot

Reply:
Yes manage users may be needed for api call to change locale if the user .  @Bonymol-aot / @abilpraju-aot  cna you guys confirm on use wrt Internationalisation

Reply:
manage-users is needed to update user locale related to internationalization @abhilash-aot

Reply:
Good catch, thanks, updated 👍